cognite/client/_api/data_modeling/instances.py

```diff
@@ -399,17 +393,14 @@ def _create_other_params(
             other_params["instanceType"] = instance_type
         return other_params
 
-    @classmethod
-    def _dump_instance_source(cls, source: ViewIdentifier | View) -> dict:
-        instance_source: ViewIdentifier
-        if isinstance(source, View):
-            instance_source = source.as_id()
-        else:
-            instance_source = source
-        return {"source": ViewId.load(instance_source).dump(camel_case=True)}
+    @staticmethod
+    def _dump_instance_source(sources: ViewIdentifier | Sequence[ViewIdentifier] | View | Sequence[View]) -> list[dict]:
+        return [
```

Comment:
What about just handling it here?
```suggestion
    def _dump_instance_source(sources: ViewIdentifier | Sequence[ViewIdentifier] | View | Sequence[View]) -> list[dict]:
        if isinstance(sources, View):
            sources = sources.as_id()
        elif isinstance(source, Sequence) and source and isinstance(source[0], View):
            sources = [source.as_id() for source in sources]
        return [
```

Reply:
➕

Reply:
Why add unnecessary code to maintain? You have written `_load_identifier` in a way that supports `View`.

Reply:
Don't want the ids module to know about views. Especially when it introduces a circular dep.

Reply:
Aight, reverted `mypy` commit and adopted the code snippet above (with 2 typos fixed) 👍 

```
mypy.....................................................................Failed
- hook id: mypy
- exit code: 1

cognite/client/_api/data_modeling/instances.py:401: error: Item "ViewId" of
"Union[ViewId, Tuple[str, str], Tuple[str, str, str], View]" has no attribute
"as_id"  [union-attr]
                sources = [src.as_id() for src in sources]
                           ^~~~~~~~~
cognite/client/_api/data_modeling/instances.py:401: error: Item
"Tuple[str, ...]" of
"Union[ViewId, Tuple[str, str], Tuple[str, str, str], View]" has no attribute
"as_id"  [union-attr]
                sources = [src.as_id() for src in sources]
                           ^~~~~~~~~
cognite/client/_api/data_modeling/instances.py:403: error: Argument 1 to
"_load_identifier" has incompatible type
"Union[ViewId, Tuple[str, str], Tuple[str, str, str], List[Union[Any, ViewId]], Sequence[Union[ViewId, Tuple[str, str], Tuple[str, str, str]]], Sequence[View]]";
expected
"Union[Union[Tuple[str, str], Tuple[str, str, str], DataModelingId, VersionedDataModelingId, NodeId, EdgeId, InstanceId], Sequence[Union[Tuple[str, str], Tuple[str, str, str], DataModelingId, VersionedDataModelingId, NodeId, EdgeId, InstanceId]]]"
 [arg-type]
    ...mp(camel_case=True)} for dct in _load_identifier(sources, "view").as_d...
                                                        ^~~~~~~
Found 3 errors in 1 file (checked 122 source files)
```